tests/tls_e2e/common/mbedtls_utility.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <mbedtls/pk.h>
+#include <mbedtls/x509_crt.h>
+#include "utility.h"
+
```

Comment:
Where does `TLS_SERVER` and `TLS_CLIENT` get defined for the Mbed TLS case?

Reply:
Existing Mbedtls code does not have TLS_SERVER and TLS_CLIENT in the LOG_INFO or LOG_ERROR statements. It is something introduced by me in the openssl tests.